bool monitor_energy
bool reset_energy

---

bool success
